### ISSUE TITLE
Adding advanced for React

### ISSUE THREAD

**Issue by Giuliano1993** [Giuliano1993/make-js-component#79]
Following the model used for VUe advanced ( PR #78 ), we can create Advanced for React to allow user to "compose" their component with the parts they prefer

**Comment by amitkroutthedev** [comment]
Can I work on this issue?

**Comment by Giuliano1993** [comment]
Hi @amitkroutthedev ! Of course you can, assigning this to you :D

**Comment by Priyanshudotdev** [comment]
@Giuliano1993 , can i add afunctional react components like shadcn ?

**Comment by Giuliano1993** [comment]
Hi @Priyanshudotdev  ! You mean calling the shadcn components installation directli from the component creation process?

**Comment by Priyanshudotdev** [comment]
No, but what I’m saying is that I can create some high-quality components. This repository currently lacks any particularly good or useful components. However, if you need the installation feature, I can figure it out.

By the way, I’m a beginner to open-source.

**Comment by Giuliano1993** [comment]
Yes the basic idea was to have a lightweight npx command to just quickly create empty components for one's projects.
 
However I find your idea very interesting! Maybe we could find a way to implement it!

If you like, open an issue about that so we can discuss it there 😁
No prob for being beginner in open-source, the important thing is try to give ones contribution and share ideas, I'd gladly help you trough the process if you wish :)

**Comment by Priyanshudotdev** [comment]
Yes, Thanks for encouragement! 😊
I’m looking forward to contributing and learning through this process!
